CMakeLists.txt

```diff
@@ -145,8 +145,18 @@ endif()
 ###############################################################################
 # Installation paths
 ###############################################################################
+option(APPEND_PROJECT_NAME_TO_INCLUDEDIR
+  "When ON headers are installed to a path ending with a folder called \
+  ${PROJECT_NAME}. This avoids include directory search order issues when \
+  overriding this package from a merged catkin, ament, or colcon workspace."
+  OFF)
+
 set(BIN_INSTALL_DIR bin/ CACHE PATH "Installation directory for binaries")
-set(INCLUDE_INSTALL_DIR include/ CACHE PATH "Installation directory for C++ headers")
+set(_include_dir "include/")
```

Comment:
We used to `unset` auxiliary variables like `_include_dir` when are no longer necessary.

Reply:
Unset `_include_dir` in 0e54e5d